modules/plink2/extract/meta.yml

```diff
@@ -0,0 +1,64 @@
+name: plink2_extract
+description: Subset plink pfiles with a text file of variant identifiers
+keywords:
+  - plink2
+  - extract
+tools:
+  - plink2:
+      description: |
+          Whole genome association analysis toolset, designed to perform a range
+          of basic, large-scale analyses in a computationally efficient manner
+      homepage: http://www.cog-genomics.org/plink/2.0/
+      documentation: http://www.cog-genomics.org/plink/2.0/general_usage
+      tool_dev_url: None
+      doi: ""
+      licence: ['GPL v3']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - pgen:
+      type: file
+      description: PLINK 2 binary genotype table
+      pattern: "*.{pgen}"
+  - psam:
+      type: file
+      description: PLINK 2 sample information file
+      pattern: "*.{psam}"
+  - pvar:
+      type: file
+      description: PLINK 2 variant information file
+      pattern: "*.{pvar}"
+  - variants:
+      type: file
+      description: A text file containing variant identifiers to keep (one per line)
+      pattern: "*.{keep}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - pgen:
+      type: file
+      description: PLINK 2 binary genotype table
```

Comment:
For each of these, extend the sentence so to distinguish from the input

Reply:
good point! are the new descriptions clearer? 40ca9ce